Modules/Scripted/EditorLib/LabelSelectDialog.py

```diff
@@ -0,0 +1,94 @@
+
+import qt
+import slicer
+
+#-----------------------------------------------------------------------------
+def _map_property(objfunc, name):
```

Comment:
May be worth noting that `objfunc` takes a `self`. The original usage AFAICT is to take something with a Qt UI (i.e. from a `.ui` file) and create a Pythonic `property` on the object owning the UI that maps to a property on a Qt object within the UI. The `objfunc` is a lambda to obtain the appropriate object from the UI.

I'm not sure `attributename` is appropriate, as that parameter refers to a property of a Qt object. (Personally I'd leave it `name` — which I guess I did when I originally wrote it :smile: — referring to the "property" of the function name.)